plugins/DBSize/spec.rb

```diff
@@ -0,0 +1,40 @@
+require_relative './plugin.rb'
+describe 'DBSize plugin' do
+  before(:all) do
+    @db_table = Iam.DB[:db_size_measurements]
+    db_key = 'Data Base Size in Bytes'
+  end
+
+  # Store method
+  describe '.store method' do
+    before(:all) do
+      DBSize.new.register
+      @cache = Cache.new(ENV['CACHE_DIR'])
+    end
+
+    before(:each) do
+      @cache.set('test_db', db_key: "123456.0", active: true)
```

Comment:
`db_key` is the same i set it in the `before(:all)` clause because it was too long line-wise
 and the 'test_db' is just a temp name for the test